cookbooks/fb_ntp/resources/windows_config.rb

```diff
@@ -0,0 +1,39 @@
+action_class do
+  def get_current_config
+    config = {}
+    section = nil
+    s = powershell_exec('w32tm /query /configuration')
```

Comment:
can/should this be moved to a ohai plugin?

Reply:
No, resources should always check the state at runtime.